README.md

```diff
@@ -151,6 +151,25 @@ class NestedIterationJob < ApplicationJob
 end
 ```
 
+
+```ruby
+class DeleteXyzJob < ApplicationJob
+  include JobIteration::Iteration
+
+  def build_enumerator(params)
+    enumerator_builder.while query_model_xyz(params).exists?
```

Comment:
While this is a valid ruby syntax, is is an intended one? I believe it was supposed to be a block
```suggestion
    enumerator_builder.while { query_model_xyz(params).exists? }
```